src/featureManager.ts

```diff
@@ -77,3 +80,9 @@ export class FeatureManager {
 interface FeatureManagerOptions {
     customFilters?: IFeatureFilter[];
 }
+
+function validateFeatureFlagFormat(featureFlag: any): void {
+    if (featureFlag.enabled !== undefined && typeof featureFlag.enabled !== "boolean") {

```

Comment:
We do support string values for "enabled" if its "true" or "false" string.

Reply:
Like these tests show:
https://github.com/Azure/AppConfiguration/blob/d0ef7d5507b9cee15893eead5861e3b1a82d3efd/docs/FeatureManagement/Samples/NoFilters.sample.json#L20-L35

Reply:
enabled is defined as boolean type in schema, which is the contract and customers should follow. I prefer not to support parsing “string” type here for now, until it’s requested by a number of JavaScript customers for some reason.

See https://github.com/Azure/AppConfiguration/issues/918

Reply:
Ok, we do support string type in the .NET provider (maybe for backwards compatibility). But it makes sense to skip it in JS if it's not needed.